client/src/components/common/input/CustomInput.js

```diff
@@ -19,6 +19,7 @@ function CustomInput({
 				name={name}
 				value={value}
 				onChange={onChange}
+				required
```

Comment:
If we add this, then we will get the browser default messages for input field on empty or invalid value, which is something we don't want.